components/scream/src/control/tests/dummy_atm_proc.hpp

```diff
@@ -67,27 +67,33 @@ class DummyProcess : public scream::AtmosphereProcess {
     FieldLayout layout_vec ( {COL,CMP,LEV}, {num_cols,2,num_levs} );
 
     if (m_dummy_type==A2G) {
-      m_input_fids.emplace("A",layout,ekat::units::m,m_grid->name());
-      m_output_fids.emplace("B",layout,ekat::units::m,m_grid->name());
-      m_output_fids.emplace("C",layout,ekat::units::m,m_grid->name());
-      m_input_fids.emplace("D",layout_vec,ekat::units::m,m_grid->name());
-      m_input_fids.emplace("E",layout,ekat::units::m,m_grid->name());
+      add_required_field("A",layout,ekat::units::m,m_grid->name());
+      add_computed_field("B",layout,ekat::units::m,m_grid->name());
+      add_computed_field("C",layout,ekat::units::m,m_grid->name());
+      add_required_field("D",layout_vec,ekat::units::m,m_grid->name());
+      add_required_field("E",layout,ekat::units::m,m_grid->name());
     } else if (m_dummy_type == G2A) {
-      m_output_fids.emplace("A",layout,ekat::units::m,m_grid->name());
+      add_computed_field("A",layout,ekat::units::m,m_grid->name());
+      add_required_group("The Group",m_grid->name());
+    } else {
+      add_updated_group("The Group",m_grid->name());
     }
   }
 
   // Register all fields in the given repo
-  void register_fields (FieldRepository<Real>& field_repo) const {
+  void register_fields (const std::map<std::string,std::shared_ptr<FieldManager<Real>>>& field_mgrs) const {
+    auto repo = field_mgrs.at(m_grid->name());
```

Comment:
Are we trying to convert all `repo` variable names to `mgr`? Not a big deal, but while we're in here...

Reply:
ah, yes, I was trying to do that. I missed this.